db/migrate/20241024094215_add_customer_name_column_to_organizations.rb

```diff
@@ -0,0 +1,9 @@
+class AddCustomerNameColumnToOrganizations < ActiveRecord::Migration[7.2]
+  def up
+    add_column :organizations, :requester_name, :string
+  end
+
+  def down
+    remove_column :organizations, :requester_name
+  end
+end
```

Comment:
This can be removed since we won't be creating the new organization until it has been vetted